.github/workflows/release.yaml

```diff
@@ -32,8 +32,13 @@ env:
 
 jobs:
 
-  build:
+  tag:
```

Comment:
Why do you need this extra job ?

Reply:
I thought that separating it in various jobs was more clean. Updated to merge all jobs into one as before